src/Integration/Handler.php

```diff
@@ -7,6 +7,8 @@
 /**
  * This class handles the state of already installed integrations.
  * It makes sure to call {@link IntegrationInterface::setupOnce} only once per integration.
+ *
+ * @internal since version 2.4
```

Comment:
Why? Isn't `final` enough?

Reply:
It isn't, marking a class as `@internal` allow us to change it however we like, included deleting it or changing the public API however we like. We cannot do it as of now anyway as the class wasn't internal before, but letting people know that it's not mean to be part of the public API is a first step towards that